pkgdb/src/pkgdb/input.cc

```diff
@@ -67,7 +73,8 @@ PkgDbInput::init()
           std::this_thread::sleep_for( DurationMillis( 250 ) );
           if ( ++retries > 100 )
             {
-              throw PkgDbException( "couldn't initialize package database" );
+              throw PkgDbException(
+                "couldn't initialize read-only package database" );
```

Comment:
suggestion (nb): worth including a reason here?
"couldn't initialize read-only package database, because it is still being initialized?" or is the reason not as clear cut?